### ISSUE TITLE
MPD validity duration

### ISSUE THREAD

**Issue by irajs** [Dash-Industry-Forum/DASH-IF-IOP#372]
A dynamic MPD SHALL conform to the constraints in this document not only at its moment of initial publishing but through the entire <dfn>MPD validity duration</dfn>, which is a period of `MPD@minimumUpdatePeriod` starting from the moment the MPD download is started by a client, unless overridden by [[#inband|in-band validity signaling]].

"Unless..." is true only if inband validity expiration event is supported by all client. Should we make it a SHALL?

**Comment by sandersaares** [comment]
Yes, further clarification on this is beneficial. I have adjusted the relevant chapter to increase understandability and remove ambiguity. Changes not yet published as I don't want to change the verison at the URL under community review but you can see them in the separate post-communityreview branch.
